### DISCUSSION TITLE
How to turn pump on every 20 minutes and keep it on for 10 minutes

### DISCUSSION THREAD

**Discussion by amoktar** [arendst/Tasmota#16050] [Q&A] [answered]
Hello 👋🏻 
I want to turn pump on every 20 minutes and keep it on for 10 min then turn it off
Thanks

**Comment by sfromis** [comment, marked as answer]
You can use rules, with RuleTimer to control the 30-minute cycle, supplemented PulseTime to control the 10-minute on-period.
